packages/vx-event/src/getXAndYFromEvent.ts

```diff
@@ -0,0 +1,22 @@
+import { EventType } from './types';
+import { isTouchEvent } from './typeGuards';
+
+const DEFAULT_POINT = { x: 0, y: 0 };
+
+export default function getXAndYFromEvent(event?: EventType) {
+  if (!event) return { ...DEFAULT_POINT };
```

Comment:
yeah this is tricky. there are many instances across packages now that use whatever is returned by this function and assume numeric/valid `x/y` values exist.

that's why I went down the path of returning numeric values, but I agree that it doesn't seem right/is misleading, given an empty event. I like the idea of returning `null`s to signify invalid coordinates, it would still allow for type safety. I think this will require a change in `@vx/point` to support `x: number | null`, tho. wdyt?

Reply:
I think no matter what, this means we have to update calls elsewhere to check for valid points. In that case I wonder if we could simply return `null` or `undefined` here instead of wrapping in a `Point` with `null` coordinates 🤔